hibernate-core/src/main/java/org/hibernate/boot/registry/selector/internal/StrategySelectorImpl.java

```diff
@@ -149,30 +141,6 @@ public StrategySelectorImpl(ClassLoaderService classLoaderService) {
 			Class<T> strategy,
 			Object strategyReference,
 			Callable<T> defaultResolver) {
```

Comment:
Removed implementations of new methods added to StrategySelector interface (that are removed below in this commit).